libpod/container_internal.go

```diff
@@ -1099,8 +1099,10 @@ func (c *Container) generateSpec(ctx context.Context) (*spec.Spec, error) {
 		}
 	}
 
-	if err := c.setupOCIHooks(ctx, &g); err != nil {
-		return nil, errors.Wrapf(err, "error setting up OCI Hooks")
+	if os.Getuid() == 0 {
```

Comment:
Is this a workaround for hooks that lack sufficient permission checks?  I'd rather patch the hooks, or wrap them with something that does permission checks, or allow users to set the hook directory (which we currently restrict to internal testing).

Reply:
the hooks we have now assume root access, that is why I've temporarily disabled them.  We can revisit this once we have hooks that can run as a non privileged user